NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -104,7 +104,9 @@ def run(self, evt, station, det):
         # for different cable delays
         times_min = []
         times_max = []
-        for iCh in det.get_channel_ids(sim_station_id):
+        if channel_ids is None:
+            channel_ids = det.get_channel_ids(sim_station_id)
+        for iCh in channel_ids:
```

Comment:
you could use the same variable as in the previous loop (`channel_id`)

Reply:
fixed